source/documentation/other-topics/rotate-tokens-keys.html.md.erb

```diff
@@ -151,15 +157,15 @@ This guide covers only secrets provided by the Cloud Platform system (your appli
 
   2. **Recycle the pods in your namespace**
   
-    Elasticache support two AUTH tokens, you can delete the pod one by one to allow new pods to pick up the new AUTH token from the kubernetes secret.
+    Elasticache support two AUTH tokens, you can delete your pods one by one to allow new pods to pick up the new AUTH token from the kubernetes secret.
```

Comment:
```suggestion
    Elasticache supports two AUTH tokens, you can delete your pods one by one to allow new pods to pick up the new AUTH token from the kubernetes secret.
```

Could they also use `kubectl rollout restart deployment/$DEPLOYMENT_NAME` https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#-em-restart-em-

I used the command before to recycle all my pods in a ns and I thought it was quite neat instead of deleting each pod?

Reply:
Something to explore as it is a nice approach, but need to be sure that different deployment methods are ok with it.